index.html

```diff
@@ -27,7 +27,7 @@
         <meta charset="UTF-8">
 
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.6.1/dat.gui.min.js"></script>
+        <script src="node_modules/dat.gui/build/dat.gui.min.js"></script>
```

Comment:
Keep old index.html in [README.md](https://github.com/iTowns/itowns2/blame/master/README.md#L29)  
> if you want to play with a demonstration, please see [http://www.itowns-project.org/itowns2/])

So change `autodeploy.sh`

and Add, if you want : dev link demonstration in readme.md